packages/integrations/vercel/test/edge-middleware.test.js

```diff
@@ -10,9 +10,4 @@ describe('Serverless prerender', () => {
 			root: './fixtures/middleware/',
 		});
 	});
-
-	it('build successful', async () => {
-		await fixture.build();
-		expect(await fixture.readFile('../.vercel/output/static/index.html')).to.be.ok;
-	});
```

Comment:
Not sure if removing this test is intentional. It looks like this test file is added by a wip commit from the base branch.

Reply:
Yes it's intentional. I had pushed this code in the destination branch when I was testing stuff. This test doesn't do anything at the moment and the assertion always fails